app/src/main/AndroidManifest.xml

```diff
@@ -25,6 +25,8 @@
         android:required="false" />
 
```

Comment:
From [this list](https://developer.android.com/guide/topics/manifest/uses-feature-element#permissions), we should also add:
```
<uses-feature
    android:name="android.hardware.microphone"
    android:required="false" />
```